### PR TITLE
Restructure hypertable how-to to fit new how-to templates

### PR DESCRIPTION
**Pull request by charislam**
# Description

Restructure hypertable how-to to fit new how-to templates.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Partial #60

### CODE REVIEW

File: timescaledb/how-to-guides/hypertables/about-hypertables.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+created.
+
+In practice, this means that the start time of your earliest chunk doesn't
+necessarily equal the earliest timestamp in your hypertable. Instead, there
+might be a time gap between the start time and the earliest timestamp. This
+doesn't affect your usual interactions with your hypertable, but might affect
+the number of chunks you see when inspecting it.
+</highlight>
+
+#### Best practices for time partitioning
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hypertables/about-hypertables.md:46]: I feel like four heading levels is a little too much. Can we bring this back a bit somehow?

**Comment by Loquacity** [comment]
I wonder if we need to split distributed hypertables out to their own thing? There's a *lot* of content here ...

**Comment by charislam** [comment]
> I wonder if we need to split distributed hypertables out to their own thing? There's a _lot_ of content here ...

Yeah, I thought about this too, this just keeps growing. I kept them in at first because there's a lot that overlaps, but I'm going to try taking them out now

**Comment by Loquacity** [comment]
I think this is looking really good now, nice work 👍

**Review by Loquacity — APPROVED** [review]

**Pull request merged**